src/viam/components/board/board.py

```diff
@@ -30,45 +30,55 @@ class Board(ComponentBase):
         RESOURCE_NAMESPACE_RDK, RESOURCE_TYPE_COMPONENT, "board"
     )
 
-    class AnalogReader:
+    class Analog:
         """
-        AnalogReader represents an analog pin reader that resides on a Board.
+        AnalogReader represents an analog pin reader or writer that resides on a Board.
         """
 
         name: str
-        """The name of the analog reader"""
+        """The name of the analog pin"""
 
         def __init__(self, name: str):
             self.name = name
 
         @abc.abstractmethod
         async def read(self, *, extra: Optional[Dict[str, Any]] = None, timeout: Optional[float] = None, **kwargs) -> int:
             """
-            Read the current value.
+            Read the current value from a reader.
 
             ::
 
                 my_board = Board.from_robot(robot=robot, name="my_board")
 
-                # Get the GPIOPin with pin number 15.
-                pin = await my_board.gpio_pin_by_name(name="15")
-
-                # Get if it is true or false that the pin is set to high.
-                duty_cycle = await pin.get_pwm()
```

Comment:
This paragraph is unrelated to my main change, but also has nothing to do with the function it's documenting. Reading an analog pin doesn't involve PWM, and I suspect this "documentation" was added via a copy-and-paste mistake.